### ISSUE TITLE
Kickstart 3.2 (Walker Rom) not in the Rom List

### ISSUE THREAD

**Issue by CypherXG** [tonioni/WinUAE#144]
Hi
Can you pls add the Walker 3.2 Kickstart Rom (Cloanto) in the rommgr.cpp?
Winuae does not list it when the rom is in the Rompath.
The CRC 32 and the SHA1 Checksum is
0x3ee21790, 0xE50DE168,0xC4A9148F,0xC303691D,0xC1CBD40A,0x53E64014.
I try to add it but it doesn't work...I don't know why...

**Comment by tonioni** [comment]
Is that unmodified original dump?

I stopped adding any modified ROMs many years ago (I'd like to remove any modified ones but it would cause problems with existing configs..) and only add ROM images that are 1:1 ROM dumps from real hardware.

**Comment by CypherXG** [comment]
Yes this is an orginal dump. You can find the Rom on the Amiga Forever CD.
It is the Amiga Walker Prototype Rom 3.2.  Build 43.1
You can see it here that this Rom is an original Rom:
https://cloanto.com/amiga/roms/
https://en.wikipedia.org/wiki/Amiga_Walker
http://www.blachford.info/computer/walker/walker.html

**Comment by midwan** [comment]
@tonioni 
It should work with something like this:

```
{ _T("KS ROM v3.2 (Walker)"), 3, 2, 43, 1, _T("Walker\0"), 524288, 248, 2, 1, ROMTYPE_KICK, 0, 0, NULL,
0x3EE21790, 0xE50DE168, 0xC4A9148F, 0xC303691D, 0xC1CBD40A, 0x53E64014 },
```
but for some reason, it doesn't in my case. I'm not sure why at the moment.
I'm using the actual CRC32 and SHA-1 checksums, and set the CPU to 2 (Walker came with an 030).
Also note that this was a Cloanto release, so it's might be using the `rom.key` for decrypting? I'm not 100% sure, and I did try both settings (`int cloanto` set to 0/1). It didn't seem to make a difference.

(Note: the id 248 I used is because I also added two more entries for the Kick 3.1.4 ROMs that recently came out from Hyperion).

**Comment by tonioni** [comment]
Obviously encrypted ROM isn't an original dump..
Cloanto roms are automatically decrypted (if key is available) and CRC32/SHA1 must be generated from decrypted ROM.

**Comment by tonioni** [comment]
I am not going to add (at least not that soon) 3.1.4 ROMs to rommgr because for too many users the ROM list practically equals "must collect them all". Also apparently those ROMs have already been modified at least once..

**Comment by CypherXG** [comment]
Ok, No problem. Kick 3.2 Walker Rom encrypted.
Here is the CRC32:
261339f8
Here is the SH1:
7a9095f1107966f90267dc4cb3c1972efb4b78a8

I hope i can help you....
Best regards
